contracts/deposit-service/ReceiverImplementation.sol

```diff
@@ -61,16 +63,17 @@ contract ReceiverImplementation is DepositServiceBase {
             return;
         }
 
-        address wrappedTokenAddress = wrappedToken();
         uint256 amount = address(this).balance;
 
         if (wrappedTokenAddress == address(0)) revert WrappedTokenNotSupported();
+        // slither-disable-next-line incorrect-equality
         if (amount == 0) revert NothingDeposited();
 
         // Wrapping the native currency and into WETH-like
         IWETH9(wrappedTokenAddress).deposit{ value: amount }();
         // Not doing safe approval as gateway will revert anyway if approval fails
         // We expect allowance to always be 0 at this point
+        // slither-disable-next-line unused-return
         IWETH9(wrappedTokenAddress).approve(gateway, amount);
```

Comment:
why not check the return value?

Reply:
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/ERC20.sol#L81